src/Main.java

```diff
@@ -1833,9 +1851,13 @@ else if (location.equals("S6S")) {
         }
         else if (location.equals("S6R")) {
             return canAccess("S6", inventory)
+				/*
+				* Downgraded S6R logic without a Glove from MINOR GLITCHES to NORMAL.
+				* You just need to break the block, exit the room, then use the other entrance to reach the chest without needing a High Walljump.
+				*/
                     && inventory.contains(Items.JUMP_BOOTS)
-                    && (difficulty >= Difficulty.HARD || inventory.contains(Items.SPIKED_HELMET))
-                    && ((difficulty >= Difficulty.S_HARD && inventory.contains(Items.SPIKED_HELMET)) || canLift(inventory));
+                    && ((difficulty > Difficulty.EASY && inventory.contains(Items.SPIKED_HELMET)) || canLift(inventory))
```

Comment:
Yes, this would be how it should go. MG was downgraded to Normal because the previous high walljump that was to be used for logic here isn't necessary since you can then just go to the left side entrance and reach the Red Chest that way. The high walljump may be faster to execute than going back through the left side entrance, but it's not necessary to get through this.